site/content/en/guides/producer/functions/container/_index.md

```diff
@@ -7,30 +7,39 @@ description: >
    Writing and running functions as containers
 ---
 
-Functions may be written as container images.  The container must run a program which:
+Functions may be written as container images. The container must run a program
+which:
 
 - Reads a ResourceList from STDIN
 - Writes a ResourceList to STDOUT
 - Exits non-0 on failure
 - Writes error messages to users on STDERR
 
-By default containers are run without network access, and without the ability to write to volumes
-outside the container.
+By default containers are run without network access, and without the ability
+to write to volumes outside the container. All local environment variables
+except the `TMPDIR` variable are loaded into the container by default.
 
-While container functions may be written in any language so long as they adhere to the io
-specification (read / write ResourceList), the [kpt-functions-sdk] provides a solution for
-writing functions using typescript.
+While container functions may be written in any language so long as they
+adhere to the [io specification] (read / write ResourceList), the
+[kpt-functions-sdk] provides a solution for writing functions using typescript
+and the [go libraries] provide utilities for writing them in golang.
 
 ## Imperative Run
```

Comment:
We're repeating the same information for both producers and consumers. There should be one place talking about ths.

Reply:
Removed mentions of writing functions from the consumer docs. It's appropriate to mention it in producer docs.